Fedoraware/TeamFortress2/TeamFortress2/Features/Menu/Menu.cpp

```diff
@@ -1538,6 +1535,8 @@ void CMenu::MenuMisc()
 		/* Column 3 */
 		if (TableColumnChild("MiscCol3"))
 		{
+			SectionTitle("Menu Customization");
+			InputKeybind("Menu key", Vars::Misc::MenuKey); HelpMarker("The key to open the menu");
```

Comment:
Change this to `InputKeybind("Menu key", Vars::Misc::MenuKey, false, true); HelpMarker("The key to open the menu");` to allow special keys